softioc/device_core.py

```diff
@@ -1,46 +1,11 @@
 from __future__ import print_function
 
-import sys
 from . import imports
 from .fields import RecordFactory
 from ctypes import *
 
 
-# Black magic lifted from six.py (http://pypi.python.org/pypi/six/) to replace
-# use of __metaclass__ for metaclass definition
-def with_metaclass(meta, *bases):
-    class metaclass(meta):
-        def __new__(cls, name, this_bases, d):
-            return meta(name, bases, d)
-    return type.__new__(metaclass, 'temporary_class', (), {})
-
-
-class InitClass(type):
-    def __new__(cls, name, bases, dict):
-        if '__init_class__' in dict:
-            dict['__init_class__'] = classmethod(dict['__init_class__'])
-        return type.__new__(cls, name, bases, dict)
-
-    def __init__(cls, name, bases, dict):
-        type.__init__(cls, name, bases, dict)
-        # Binds self.__super.method to the appropriate superclass method
-        setattr(cls, '_%s__super' % name.lstrip('_'), super(cls))
-        # Binds cls.__super_cls().method to the appropriate superclass
-        # class method.  Unfortunately the .__super form doesn't work
-        # with class methods, only instance methods.
-        setattr(
-            cls, '_%s__super_cls' % name,
-            classmethod(lambda child: super(cls, child)))
-        # Finally call the class initialisatio nmethod.
-        cls.__init_class__()
-
-class DeviceCommon(with_metaclass(InitClass)):
-    '''Adds support for an __init_class__ method called when the class or any
-    of its subclasses is constructed.  Also adds auto-super functionality
-    (see iocbuilder.support.autosuper).'''
-
-    def __init_class__(cls): pass
-
+class DeviceCommon():
```

Comment:
Don't think the brackets `()` are needed here, are they?

Reply:
Removed in 9b243807d473fe1daa1b067fc310eb7e870eb09d